tests/parm/ice_in_template

```diff
@@ -1,27 +1,27 @@
 &setup_nml
     days_per_year  = 365
     use_leap_years = .true.
-    year_init      = YEAR_INIT
-    month_init     = MONTH_INIT
-    day_init       = DAY_INIT
-    dt             = DT_CICE
+    year_init      = @[SYEAR]
+    month_init     = @[SMONTH]
+    day_init       = @[SDAY]
+    dt             = @[DT_CICE]
     npt            = 999
     ndtd           = 1
-    runtype        = 'CICERUNTYPE'
-    runid          = 'RUNID'
+    runtype        = '@[CICERUNTYPE]'
+    runid          = '@[RUNID]'
     ice_ic         = 'cice_model.res.nc'
     restart        = .true.
-    restart_ext    = RESTART_EXT
-    use_restart_time = USE_RESTART_TIME
+    restart_ext    = .false.
```

Comment:
Shouldn't this be:
```suggestion
    restart_ext    = @[RESTART_EXT]
```

Reply:
I decided to remove this as a configurable option since we now always have it set false by default always.